### DISCUSSION TITLE
Auraflow?

### DISCUSSION THREAD

**Discussion by tobbe628** [lllyasviel/stable-diffusion-webui-forge#1436] [General]
With Pony V7 dropping on Auraflow in the future.
https://civitai.com/articles/6309

Can we expect to be able to use Auraflow on Forge?

Thanks

**Comment by lllyasviel** [comment]
Thanks for the post! 

I do not know if Forge will add more models, but below is my thoughts about the post if someone finds it useful.

----

I read it a bit – although the link mentions Flux license, the real reason behind that is probably computation requirement to train the very big Flux.

But I recommend to wait a bit because it is super likely that many SD training scripts will go through large changes recently to adapt to LLM training strategies like iterative QLoRA merging during training, quantized gradient backpropagation, partial gradient graph freezing, etc. 

In LLM, almost no one use current SD community’s methods train models. To get an idea how people train LLM, one can take a look at https://github.com/unslothai/unsloth

Once some more techs are implemented, people will get as large batch size as previous SDXL when training Flux, and many model authors may get some new plans.

**Reply by BNP1111** [reply]
Thank you for telling us something(or the truth) based on academic rigor, you are trustworthy
